.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question via Jaeger Community Support (online chat)
+    url: https://cloud-native.slack.com/archives/CGG7NFUJ3.
+    about: Please ask and answer questions here for faster response. If you don't have an account, signup at: https://slack.com/intl/en-au/get-started.
```

Comment:
The sign up url is https://slack.cncf.is

Reply:
Thanks, updated the URL.